test/fixtures/before.js

```diff
@@ -0,0 +1,7 @@
+import _ from "lodash";
+
+import { map as map2 } from "lodash";
+
+import debounce from "lodash/debounce";
+
+import throttle from "lodash-es";
```

Comment:
It can be delete when after you check

just run below command at your test env
```
npx hyesungoh-test-codemod test/fixtures # or anyplace you can check it
```

> [!Note]
> `hyesungoh-test-codemod` is running same code in this PR
> this lib will be delete after this pr closed